microsoft-edge/webview2/how-to/winrt-from-js.md

```diff
@@ -0,0 +1,336 @@
+---
+title: Call native-side WinRT code from web-side code
+description: How to pass WinRT host objects to JavaScript using the AddHostObjectToScript API for WebView2 apps.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 05/02/2022
+---
+# Call native-side WinRT code from web-side code
+
+Your web-side JavaScript code can access the methods and properties of your WinRT host app, by using the _WebView2 WinRT JS Projection tool_.  (For short, this is sometimes called the _WV2WinRT_ tool.)  For more information about why you'd want to do this, see the introduction of [Call native-side code from web-side code](hostobject.md).
+
+This article walks you through the following main steps:
+
+*  Create a **WinRTAdapter** project especially to contain the _WebView2 WinRT JS Projection tool_.
```

Comment:
Is "contain" the best word here? The project does use the tool but it doesn't _contain_ it in the sense of having any binary linked to the output.

In pure technical terms, if that helps, the project builds a library out of code generated by running the tool. This generated code allows WinRT APIs to be exposed in a WebView.